node/node.go

```diff
@@ -10,12 +10,12 @@ import (
 	"github.com/libp2p/go-libp2p/core/crypto"
 	"go.uber.org/multierr"
 
+	abcicli "github.com/tendermint/tendermint/abci/client"
```

Comment:
Nit: I don't like the abbreviation `abcicli` as it makes me think it's related to a command line interface (CLI). I'd prefer either `abciclient`

Reply:
If following strict Go style we could also use `ac`

Reply:
For some reason I thought it's the way it is used in cosmos-sdk... but it's not, so I fixed to `abciclient`.